pyproject.toml

```diff
@@ -71,12 +71,14 @@ trace_server = [
   "google-cloud-storage>=2.7.0",
   # LLM Support
   "litellm>=1.36.1",
+   # OTEL tracing support
+  "opentelemetry-proto>=1.12.0",
 ]
 trace_server_tests = [
   # BYOB - S3
   "moto[s3]>=5.0.0",
 ]
-docs = ["playwright", "lazydocs", "nbformat", "nbconvert"]
+docs = ["playwright", "lazydocs", "nbformat", "nbconvert", "opentelemetry-proto>=1.12.0"]
```

Comment:
needed for docs?!

Reply:
Unfortunately yes. I asked Andrew about this and seems its a bug. When the dep is only in trace server, the docs job fails to run in CI since trace_server_interface includes the proto import. 

Docgen for trace server should require all the trace_server deps but it doesn't. This was better than making the proto package a top level requirement which was the other fix that worked

Reply:
oh, you could just make docs depend on server.... but i got it